### ISSUE TITLE
the profiler does not detect my application on ubuntu22.04 

### ISSUE THREAD

**Issue by tylerfan** [GPUOpen-Tools/radeon_gpu_profiler#93]
it detects my GPU in the system section which is  Radeon RX 5500M which i think is in the supported GPU list. The application status is always offline.

is there a way to launch the program to profile from the profiler itself?

**Comment by chesik-amd** [comment]
One reason this can happen is if application your end ends up using the RADV driver instead of the AMD Vulkan driver. On recent Ubuntu versions, the open source RADV driver is installed by default and a Vulkan app may end up using the RADV ICD instead of the AMD Vulkan ICD.  If this is what is happening on your system, setting the following environment variable should help:  VK_ICD_FILENAMES=/etc/vulkan/icd.d/amd_icd64.json.

https://github.com/GPUOpen-Tools/radeon_gpu_profiler/blob/58b0dd90c62375536e31665c2d092ef607c5139a/RELEASE_NOTES.txt#L48C436-L48C485

**Comment by nazar-pc** [comment]
Now that AMD discontinued their Vulkan driver and RADV is the only thing going forward, any plans to support it?

I just tried on 2025-07-01-1408 release on Ubuntu 24.04 and it still doesn't see my apps in the list. Does it still not work with amdgpu (which is in the kernel) + RADV?

**Comment by chesik-amd** [comment]
See this note in the v2.6 Release Notes: https://github.com/GPUOpen-Tools/radeon_gpu_profiler/blob/v2.6/RELEASE_NOTES.txt#L34

You can capture data when running on RADV by using environment variables, but capturing using Radeon Developer Panel is not supported. The relevant environment variables are documented here: https://docs.mesa3d.org/envvars.html#envvar-MESA_VK_TRACE

**Comment by nazar-pc** [comment]
Yes, that is the only way I found that actually works. But usability of it could have been a lot better.
